modules/apps/jaggery/emm/src/emm/units/user-listing/user-listing.js

```diff
@@ -1,13 +1,17 @@
 function onRequest(context) {
     var userModule = require("/modules/user.js")["userModule"];
     context["permissions"] = userModule.getUIPermissions();
-    if(userModule.isAuthorized("/permission/admin/device-mgt/users/delete")){
+    if (userModule.isAuthorized("/permission/admin/device-mgt/users/delete")){
```

Comment:
format the if clause at its end -> if () {...

Reply:
Fixed